cmd/bcpd/app/app_test.go

```diff
@@ -300,8 +315,8 @@ func appStateGenesis(t require.TestingT, contracts []*contract) string {
 			dt{"ticker": "ETH", "name": "Smells like ethereum"},
 			dt{"ticker": "FRNK", "name": "Frankie"},
 		},
-		"gconf": dt{
-			"cash:collector_address": "66616b652d636f6c6c6563746f722d61646472657373",
+		"cashconf": cash.Configuration{
```

Comment:
Oops, this still has old, invalid genesis format. I guess this is never run in test code?